spec/core.md

```diff
@@ -216,40 +218,47 @@ DID-related documents have been copied.
 
 ### DID Method Operations
 
-#### Create (Register)
-
-Creating a DID involves these steps:
-
-- Choose the web URL where the DID document for the DID will be published, excluding
-  the last element that will be the AID, once defined.
-- Create a KERI AID and add the appropriate KERI events that will (at least)
-  produce the DID document. That process is described in the [DID Document from KERI Events](#did-document-from-keri-events)
-  generation section of this specification.
-- Add the AID as the last element of the web URL for the DID.
-- Derive the DID document by processing the [[ref: KERI event stream]].
-- Create the AID folder on the web server at the selected location, and place
-the DID document and the [[ref: KERI event stream]] file into that folder.
-
-Of course, the web server that serves the files when asked might be a simple
-file server (as implied above) or an active component that retrieves them
-dynamically. Further, the publisher of the files placed on the web can use
-capabilities like [CDNs] to distribute the files.
+#### Create
+
+Creating a `did:webs` DID involves the following steps:
+
+1. Choose the web URL where the DID document for the DID will be published, excluding
+   the last element that will be the AID, once defined.
+2. Create a KERI AID and add the appropriate KERI events that will correspond to
+   properties of the DID document, such as verification methods and service endpoints.
+3. Add the AID as the last element of the web URL for the DID.
+4. Derive the `did:webs` [[ref: DID document]] by processing the [[ref: KERI event stream]]
+   according to section [DID Document from KERI Events](#did-document-from-keri-events).
+5. Transform the derived `did:webs` DID document to the corresponding `did:web` DID document according
+   to section [Transformation to did:web DID Document](#transformation-to-didweb-did-document).
+6. Create the AID folder on the web server at the selected location, and place
+   the `did:web` DID document resource (`did.json`) and the [[ref: KERI event stream]] resource (`did.keri`)
```

Comment:
Sam made a good case for this to be .cesr not .keri because the IANA mime-type for cesr is going to be probationarily registered and eventually officially registered. All KEL/TEL/OOBI/etc resources are effectively CESR Streams so I believe we've settled on .cesr type.

Reply:
If you prefer I can do a pass over the whole document to update KERI event stream to CESR event stream and references to .keri to .cesr. I'm flexible.